src/main/java/edu/greenblitz/tobyDetermined/RobotMap.java

```diff
@@ -307,20 +321,23 @@ public static class elbow {
 
 
             public static final int CURRENT_LIMIT = 40;
+
+            public static final int ABSOLUTE_ENCODER_CHANNEL = 1;
         }
     }
 
-    public static class Intake{
-		public static final int ROLLER_ID = 0;
-		public static final boolean INVERTED = false;
-		public static final double RAMP_RATE = 0.1;
-		public static final double DEFAULT_POWER = 1;
-		public static final int CURRENT_LIMIT = 40;
-		public static final double ROLL_INSIDE_POWER = 0.5;
-		
-		public static class Solenoid {
-			public static final int FORWARD_PORT = 1;
-			public static final int REVERSE_PORT = 0;
-		}
-	}
+    public static class Intake {
+        public static final int ROLLER_ID = 0;
+        public static final boolean INVERTED = false;
```

Comment:
this should use sparkmax config object
why not use default ramp rate and voltage compansation